test/enigma_test.rb

```diff
@@ -31,8 +35,8 @@ def test_it_can_decipher
   end
 
   def test_it_ignores_non_alphabet_characters
-    assert_equal "keder ohulw!", @enigma.cipher("HELLO WORLD!", [3, 27, 73, 20])
-    assert_equal "hello world!", @enigma.decipher("keder ohulw!", [3, 27, 73, 20])
+    assert_equal "keder ohulw!&", @enigma.cipher("HELLO WORLD!&", [3, 27, 73, 20])
+    assert_equal "hello world!&", @enigma.decipher("keder ohulw!&", [3, 27, 73, 20])
   end
```

Comment:
I wanted to make sure it would check for more than just "!" so I updated the test to include "&"